client/components/data/query-theme-filters/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestThemeFilters } from 'state/themes/actions';
+
+export class QueryThemeFilters extends Component {
+	static propTypes = {
+		requestThemeFilters: PropTypes.func.isRequired,
+	}
+
+	componentWillMount() {
```

Comment:
I think we normally use `componentDidMount`. `WillMount` would run on the server IIRC, but at too late a stage for SSR, so we avoid it entirely there.